zipline/data/data_portal.py

```diff
@@ -232,15 +240,23 @@ def highs(self, assets, dt):
                     else:
                         after_last = pd.Timestamp(
                             last_visited_dt + self._one_min, tz='UTC')
-                        window = self._minute_reader.unadjusted_window(
-                            ['high'], after_last, dt, [asset])
+                        window = self._minute_reader.load_raw_arrays(
+                            ['high'],
+                            after_last,
+                            dt,
+                            [asset],
+                        )[0].T
```

Comment:
Just pattern matching here, but why do some of these changes include a transposition, but not others?